openedx/features/course_experience/templates/course_experience/course-outline-fragment-new.html

```diff
@@ -14,7 +14,17 @@
 
 <main role="main" class="course-outline-visualprogress" id="main" tabindex="-1">
     % if blocks.get('children'):
-        <ol class="block-tree accordion" role="presentation">
+        <button class="btn btn-primary"
+                id="expand-collapse-outline-all-button"
+                aria-expanded="false"
+                aria-controls="course-outline-block-tree"
+                >
+          <span id="expand-collapse-outline-all-span">${_("Expand All")}</span>
```

Comment:
I'm almost certain you could actually get rid of the span here, and just have text within the button. Then, you can access the text in your javascript with ```document.querySelector('#expand-collapse-outline-all-button').innerText```.

Reply:
I think Tyler told me something about having the span being the right thing to do for accessibility reasons.